src/Types/SingleOrMultiple.ts

```diff
@@ -0,0 +1 @@
+export type SingleOrMultiple<T> = T | T[];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5274115)